handlers/action-handler.go

```diff
@@ -3,71 +3,86 @@ package handlers
 import (
 	"encoding/json"
 	"fmt"
+	"google.golang.org/api/chat/v1"
 	"log"
```

Comment:
one improvement - slog with JSONHandler. Structured logging with ease and widely used. 

https://www.reddit.com/r/golang/comments/1i8timw/logging_in_golang_libraries/
https://betterstack.com/community/guides/logging/logging-in-go/

Reply:
Thank you for the suggestion! It does resemble console logging in JavaScript, which is exactly the behavior I was aiming for. I’ll make the necessary adjustments to use slog with JSONHandler as suggested.